src/pagingscrollview/src/NIPagingScrollView.m

```diff
@@ -694,6 +722,20 @@ - (void)setPageMargin:(CGFloat)pageMargin {
   [self setNeedsLayout];
 }
 
+- (void)setPageInset:(CGFloat)pageInset {
+  _pageInset = pageInset;
+  _scrollView.frame = [self frameForPagingScrollView];
```

Comment:
Funny, I had one originally.  I got some comments about it so removed it.  But it's easy to add back ;)